extensions/wikia/AssetsManager/config.php

```diff
@@ -2460,7 +2460,7 @@
 	'type' => AssetsManager::TYPE_JS,
 	'skin' => [ 'oasis' ],
 	'assets' => [
-		'//extensions/wikia/CategoryPage3/scripts/alphabet-shortcuts.js',
+		'//extensions/wikia/CategoryPage3/scripts/decode-redlink-href.js',
```

Comment:
We have an asset group called `category_page3_alphabet_shortcuts_js` that loads `decode-redlink-href.js` which is in `/extensions/wikia/CategoryPage3` and handles `a[data-uncrawlable-url]`. It's very confusing, let's clean it up.
My suggestion:
- Move this file to `/extensions/wikia/SEOTweaks/scripts`
- Rename it to `uncrawlable-urls.js`
- Change the asset group name to `seo_tweaks_uncrawlable_urls`